app/viewmodels/journeys/taskList/NationalInsuranceContributionsViewModel.scala

```diff
@@ -26,22 +28,27 @@ import viewmodels.journeys.{SummaryListCYA, getJourneyStatus}
 
 object NationalInsuranceContributionsViewModel {
 
-  def buildSummaryList(nationalInsuranceStatuses: List[JourneyNameAndStatus])(implicit messages: Messages): SummaryList = {
+  def buildSummaryList(nationalInsuranceStatuses: List[JourneyNameAndStatus], taxYear: TaxYear)(implicit messages: Messages): SummaryList = {
 
-    val nicRow = buildRow(NationalInsuranceContributions, nationalInsuranceStatuses, dependentJourneyIsFinishedForClickableLink = false)
+    val isAdjustmentsAnswered = JourneyStatus.getJourneyStatus(Adjustments, nationalInsuranceStatuses).isCompleted
+    val nicRow = buildRow(
+      NationalInsuranceContributions,
+      nationalInsuranceStatuses,
+      dependentJourneyIsFinishedForClickableLink = isAdjustmentsAnswered)(messages, taxYear)
 
     val rows: List[SummaryListRow] =
       List(nicRow)
 
     SummaryListCYA.summaryList(rows)
   }
 
-  private def buildRow(journey: Journey, nationalInsuranceStatuses: List[JourneyNameAndStatus], dependentJourneyIsFinishedForClickableLink: Boolean)(
-      implicit messages: Messages): SummaryListRow = {
+  private def buildRow(journey: Journey,
+                       nationalInsuranceStatuses: List[JourneyNameAndStatus],
+                       dependentJourneyIsFinishedForClickableLink: Boolean)(implicit messages: Messages, taxYear: TaxYear): SummaryListRow = {
     val status: JourneyStatus = getJourneyStatus(journey, dependentJourneyIsFinishedForClickableLink)(nationalInsuranceStatuses)
     val keyString             = messages(s"journeys.$journey")
     val href = journey match {
-      case NationalInsuranceContributions => "#"
+      case NationalInsuranceContributions => nics.routes.Class2NICsController.onPageLoad(taxYear, NormalMode).url
```

Comment:
Since we know which journey this method will make, we don't need to pass in the journey.
the keyString can just be `messages(s"journeys.$NationalInsuranceContributions")` and we don't need to match the journey for the href.

Reply:
Done :)